pom.xml

```diff
@@ -7,6 +7,15 @@
     <groupId>mate-academy</groupId>
     <artifactId>web-intro</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <packaging>war</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>3.0-alpha-1</version>
```

Comment:
it's better not to use alpha versions, they may be unstable. I suggest we downgrade version to latest stable